frontend/src/App.js

```diff
@@ -1,12 +1,20 @@
-import { BrowserRouter as Router } from "react-router-dom";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import "./index.css";
 import Navbar from "./Components/Navbar";
 import MiniNav from "./Components/MiniNavbar";
+import Home from "./Pages/Home";
```

Comment:
Maintain an `index.js` at the root level of components and pages folder and export each component through this file. The imports will be much cleaner and maintainable

Reply:
There was only one component in it. That's why I have exported as default.